Makefile

```diff
@@ -25,15 +38,16 @@ else
 		JAVA_HOME := $(shell /usr/libexec/java_home)
 	endif
 endif
-JAVA_PLATFORM := $(JAVA_HOME)/include/$(LOWERCASE_OS)
 
+# Rely on the current operating system to decide which JNI headers to use, and
+# for one Python flag. At the moment, this means that Android builds of rubicon-java
+# must be done on a Linux host.
```

Comment:
I'm not completely clear on why this is true.

Reply:
My thinking is we need the JNI headers (*.h) for the platform we're building for.

The JDK only bundles the JNI headers for the platform it's running on.

Happy to explain more; let me know if that answers things.